pkg/runtime/tags.go

```diff
@@ -0,0 +1,69 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package runtime
+
+import (
+	acktypes "github.com/aws/aws-controllers-k8s/pkg/types"
+	"strings"
+	"time"
+
+	ackconfig "github.com/aws/aws-controllers-k8s/pkg/config"
```

Comment:
nit: Import should preferably be organized in groups. https://github.com/golang/go/wiki/CodeReviewComments\#imports

Reply:
Organized imports.